src/content/fr/2020/performance.md

```diff
@@ -0,0 +1,388 @@
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=1135415956&format=interactive",
+  width="645",
+  height="792",
+  sheets_gid="1440255644",
+  sql_file="web_vitals_by_country.sql"
+  )
+}}
+
+La comparaison des données géographiques du TTFB de cette année avec [les résultats de 2019](../2019/performance#ttfb-by-geo) montre une fois de plus que les sites web sont plus rapides, mais comme pour le FCP, les seuils ont changé. Auparavant, nous considérions les TTFB inférieurs à 200 ms comme rapides et supérieurs à 1000 ms comme lents. En 2020, la TTFB en dessous de 500 ms est bonne et au-dessus de 1500 ms, mauvaise. Des changements aussi importants dans la classification peuvent expliquer que nous observons des changements significatifs, comme une augmentation de 36 % des bonnes expériences de sites web en République de Corée ou de 22 % à Taïwan. Dans l’ensemble, nous observons toujours les mêmes régions en tête, telles que l’Asie-Pacifique et certaines régions européennes.
+
+### TTFB par type de connexion
+
+{{ figure_markup(
+  image="performance-ttfb-by-connection-type.png",
+  caption="Performances agrégées du TTFB réparties par type de connexion.",
+  description="Graphique à barres montrant que le TTFB est fortement influencé par le type de connexion avec seulement 21 % et 22 % de bonnes expériences sur la 4G et hors ligne, respectivement. Les autres types de connexion ne fournissent pratiquement aucune bonne expérience de TTFB (à l’exception de 1 % sur la 3G).",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3ukFCTxx__dTBLwDQ3K_hCtXdLRGOGUYdR_TO043n_aCTdcwkxPUku9Qfoal6BESiV5RMtd0nEbqT/pubchart?oid=810992122&format=interactive",
+  sheets_gid="306222260",
+  sql_file="web_vitals_by_ect.sql"
+  )
+}}
+
+Le TTFB est affecté par la latence du réseau et le type de connexion. Plus la latence est élevée et plus la connexion est lente, plus les mesures de la TTFB sont mauvaises, comme nous pouvons l’observer ci-dessus. Même sur les connexions mobiles considérées comme rapides (4G), seuls 21 % des sites web ont un TTFB rapide. Il n’y a pratiquement aucun site classé comme rapide en dessous de la 4G.
+
+Si on examine [les vitesses de téléphonie mobile dans le monde pour la période décembre 2018-novembre 2019](https://www.speedtest.net/insights/blog/content/images/2020/02/Ookla_Mobile-Speeds-Poster_2020.png), on constate que, dans le monde, les connexions mobiles ne sont pas à haut débit. Ces vitesses et normes technologiques pour les réseaux cellulaires (comme la 5G) ne sont pas réparties de manière égale et affectent le TTFB. Par exemple, [voir cette carte des réseaux au Nigeria](https://www.mobilecoveragemaps.com/map_ng#7/8.744/7.670) - la plupart des régions du pays ont une couverture 2G et 3G, avec une faible portée en 4G.
```

Comment:
Should we change both these links to HTML links to add `hreflang="en"` attributes.

Reply:
Oh yeah, thanks, done 👍